src/includes/LatLng.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Take comma seperated latlng and convert into array.
+ * @param {string} latlng string.
+ * @return {Array} coordinate.
+ */
+function latLngFilter(latlng) {
```

Comment:
rename function as its not a filter maybe 'getCoordinate'

Reply:
done